deepchem/models/tests/test_graph_models.py

```diff
@@ -353,3 +346,17 @@ def test_dtnn_regression_model():
   pred = model.predict(dataset)
   mean_rel_error = np.mean(np.abs(1 - pred / y))
   assert mean_rel_error < 0.1
+
+
+@pytest.mark.tensorflow
+def test_graph_predict():
+  model = dc.models.GraphConvModel(np.random.randint(1, 100),
+                                   batch_size=100,
+                                   mode='classification')
+
+  model = dc.models.GraphConvModel(12, batch_size=50, mode='classification')
+  mols = ["CCCCC", "CCCCCCCCC"]
+  y = [0, 1]
+  feat = dc.feat.ConvMolFeaturizer()
+  X = feat.featurize(mols)
+  assert (model.predict(dc.data.NumpyDataset(X, y))).all() == True
```

Comment:
Should this be
```
assert model.predict(dc.data.NumpyDataset(X))
```
If you want to check whether this works without tasks being labeled
```
assert model.predict(dc.data.NumpyDataset(X, y))
```
but that's a different check